### DISCUSSION TITLE
is Trumbowyg still alive ?

### DISCUSSION THREAD

**Discussion by 5050** [Alex-D/Trumbowyg#1489] [General]
Hi 

Does anyone have info about status of Trumbowyg and/or Alex-D ?

I've tried to find a contact on the web (or in France from where I am) but both his personals web sites that i have found URLs have gone, His last visibles posts on facebook, SoundCloud or even X are from 2020...
Any info or contact to Alex-D would be welcome !

It would be sad that the whole nice project would be abandonned.

**Comment by Alex-D** [comment]
Hey!

I'm still alive, and I'm working on my house... Not an open source project but still takes a lot of time :p

I still plan to work on a new version one day... But for now I did not find the time and energy needed :/

Have a nice day :)

**Reply by mkrecek234** [reply]
This is very good to hear - and have a lot of success with your house 😃 May I ask for one hint? With macOS Sequoia, the line-break <br> (was Ctrl-Return before), no longer works. Any hint how to implement Ahift-Entwr in Trumbowyg also for Safari? In Chrome it atill works with shift-Enter for <br> - greatest editor ever, thank you for this great piece of code!!

**Reply by Alex-D** [reply]
To be honest, I don't know because it always be native, not custom in Trumbowyg, so I'm surprised that it didn't work

If you want to recreate that, you will need to add event listener with ctrl modifier and prevent default, then just add `<br>`? :D

**Reply by mkrecek234** [reply]
> To be honest, I don't know because it always be native, not custom in Trumbowyg, so I'm surprised that it didn't work
> 
> 
> 
> If you want to recreate that, you will need to add event listener with ctrl modifier and prevent default, then just add `<br>`? :D

Sorry to disturb you - could you give me a hint which event (on what DOM element) I have to capture to force a <br> when pressing Shift-Enter? I could not figure it out 🤔
